patch/secure_boot_kernel_config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Can just `/bin/sh` be used?

Reply:
maybe sh can be use, but I just tested it with bash, in general in sonic-slave "sh" is a soft link to "dash".
`lrwxrwxrwx 1 root root 4 Oct 24 00:00 /bin/sh -> dash
`
And most of the scripts are using bash, so I think is fine.